content/en/blog/2023-02-01-january-update/index.md

```diff
@@ -0,0 +1,384 @@
+
+### Security news
+
+### Flagger 1.27.0
+
+This release comes with support for [Apache APISIX](https://apisix.apache.org/).
+For more details see [the tutorial](/flagger/tutorials/apisix-progressive-delivery).
+
+### Flux Ecosystem
+
+<!--
+
+If you add entries to this subsection, please don't use "we" as it gets
+confusing from which perspective the whole blog post is written and who
+"we" is. The whole post is meant to be from the perspective of the Flux
+community. Better to write:]
+
+- "Since the new release of X, it supports Y." or
+- "Team lead X says: 'we have put a lot of effort into Y and are
+   really proud of the performance results' ..." or
+- "The team has been working on ..."
+
+-->
+
+#### Flux Subsystem for Argo
+
+[Flamingo](https://github.com/flux-subsystem-argo/flamingo) is a tool that
+combines Flux and Argo CD to provide the best of both worlds for
+implementing GitOps on Kubernetes clusters. With Flamingo, you can:
+
+- Automate the deployment of your applications to Kubernetes clusters and
+  benefit from the improved collaboration and deployment speed and
+  reliability that GitOps offers.
+- Enjoy a seamless and integrated experience for managing deployments,
+  with the automation capabilities of Flux embedded inside the
+  user-friendly interface of Argo CD.
+- Take advantage of additional features and capabilities that are not
+  available in either Flux or Argo CD individually, such as the robust Helm
+  support from Flux, Flux OCI Repository, Weave GitOps Terraform Controller
+  for Infrastructure as Code, Weave Policy Engine, or Argo CD
+  `ApplicationSet` for Flux-managed resources.
+
+In recent releases, the team updated Flamingo to support Flux v0.38 and Argo
+CD v2.5.7, v2.4.19 and v2.3.13. Please note that Argo CD v2.2 will not be
+supported and updated by Flamingo anymore.
+
+|Flux  | Argo CD | Image
+|:----:|:-------:|---------------------------
+|v0.38 | v2.5    | v2.5.7-fl.3-main-b0b6148f
+|v0.38 | v2.4    | v2.4.19-fl.3-main-b0b6148f
+|v0.38 | v2.3    | v2.3.13-fl.3-main-b0b6148f
+|v0.37 | v2.2    | v2.2.16-fl.3-main-2bba0ae6
+
+#### Terraform-controller
+
+The [tf-controller](https://github.com/weaveworks/tf-controller) team
+is currently working on getting [the new release
+v0.14](https://github.com/weaveworks/tf-controller/issues/344) out.
+They are updating the Terraform binary to version 1.3.7 and the Flux tool
+to version 0.38. Additionally, they are fixing the Helm chart and enabling
+the parallelism option for the apply stage. They are currently at release
+candidate v0.14.0-rc.2 with the new Helm chart version 0.10.0. Please stay
+tuned for further updates.
+
+#### Weave GitOps
+
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.15.0>
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.14.1>
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.13.0>
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.12.0>
+
```

Comment:
@makkes @josefaworks Do you know who could help writing a bit of a Dec+Jan summary about what happened in Weave GitOps Core and Enterprise?

Here are past examples:

- https://fluxcd.io/blog/2022/12/november-2022-update/#flux-ecosystem
- https://fluxcd.io/blog/2022/11/october-2022-update/#flux-ecosystem
- https://fluxcd.io/blog/2022/10/september-2022-update/#flux-ecosystem

Reply:
:pray: